ReactiveXComponent/ReactiveXComponent.csproj

```diff
@@ -13,9 +14,11 @@
   <ItemGroup>
     <PackageReference Include="ILRepack.MSBuild.Task" Version="2.0.13" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
-    <PackageReference Include="RabbitMQ.Client" Version="5.1.0" />
+    <PackageReference Include="RabbitMQ.Client" Version="4.1.3" />

```

Comment:
Why downgrading the version of RabbitMQ.client?

Reply:
We experienced some issues in runtime yesterday. So I suggested to add references to the same packages as the old version (before .Net Core) and with the same versions. I wanted to avoid any breaks which can be a consequence of upgrading some packages. XComponent runtime also uses this version of the RabbitMQ client (4.1.3).